docs/.vitepress/theme/index.ts

```diff
@@ -1,9 +1,8 @@
 import ElementPlus from 'element-plus'
 
-import 'uno.css'
-
 import VPApp, { globals, NotFound } from '../vitepress'
 import { define } from '../utils/types'
+import 'uno.css'
```

Comment:
**Suggestion:** can we not include unrelated code?

Reply:
No, we should fix style